app/main.py

```diff
@@ -33,4 +33,21 @@ def extract_text(url: HttpUrl = Query(..., description="The URL to extract text
 
     soup = BeautifulSoup(response.content, "html.parser")
     text = soup.get_text(separator="\n", strip=True)
-    return {"text": text}
+
+    # Extract images using the utility method
+    images = extract_images(soup)
+
+    return {"text": text, "images": images}
+
+def extract_images(soup):
```

Comment:
Put in a file called `scraping.py`

Reply:
Thank you for the feedback. I will move the `extract_images` function to a new file called `scraping.py` and update the import in `app/main.py` accordingly. I'll update the PR.